python/setup.py

```diff
@@ -155,6 +153,8 @@
 
 else:
     libs.append('cumlprims')
+    libs.append('cumlcomms')
+    libs.append('nccl')
```

Comment:
@dantegd this is as per our offline discussion about making `--singlegpu` option to not expect cumlcomms and nccl libraries to be installed.